examples/dockerdev/docker-compose.yml

```diff
@@ -27,13 +27,14 @@ x-backend: &backend
     - .:/app:cached
     # or for selinux: https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
     # - .:/app:cached,z
-    - rails_cache:/app/tmp/cache
-    - bundle:/usr/local/bundle
-    - node_modules:/app/node_modules
-    - packs:/app/public/packs
-    - packs-test:/app/public/packs-test
-    - .dockerdev/.psqlrc:/root/.psqlrc:ro
-    - .dockerdev/.bashrc:/root/.bashrc:ro
+    - rails_cache:/app/tmp/cache:delegated
+    - bundle:/usr/local/bundle:delegated
+    - node_modules:/app/node_modules:delegated
+    - packs:/app/public/packs:delegated
+    - packs-test:/app/public/packs-test:delegated
```

Comment:
Doesn't `delegated`,`cached` affect only volumes mounted from the host system? All of these "live" within a Docker.